scripts/bnsd/bnsd_app.sh

```diff
@@ -16,6 +16,7 @@ if [ ! -d "${BNSD_DIR}" ]; then
 fi
 
 exec docker run --user="$UID" \
+  --name "bnsd-app" \
```

Comment:
add `--rm` there for better cleanup

Reply:
Hmm, I now changed the stop script to delete the container. We should be consistent. Any good reasons for one or the other solution?

Reply:
Postponed as this affects all containers we start: https://github.com/iov-one/iov-core/issues/748